.github/workflows/buildAndTest.yml

```diff
@@ -27,6 +27,10 @@ jobs:
       with:
         java-version: 17
         distribution: temurin
+    - name: Set up Maven
+      uses: stCarolas/setup-maven@d6af6abeda15e98926a57b5aa970a96bb37f97d1 # v5
+      with:
+        maven-version: 3.9.9
```

Comment:
I would like to update the Maven version via using Java 21 in the setup-java task as the default build Java version transitively instead of introducing a new third-party GH action dependency here. Is there any reason to switch from the Maven version that is shipped with setup-java right now to justify doing this change?

Reply:
As far as I can tell, the Maven version installed by default it defined by the runner-image:
https://github.com/actions/runner-images/blob/ubuntu22/20241015.1/images/ubuntu/Ubuntu2204-Readme.md

I'm not aware that the setup-java action configures the Maven version, although it provides a Maven-cache and support for the maven tool-chain.
Currently an update of Maven is not necessary, but it might become necessary if you update for example Tycho (I'm not exactly sure about the requirements of the latest Tycho release.
But of course Maven could also just be updated explicitly if it's necessary.

Reply:
Looks like https://github.com/kieler/KLighD/pull/219 passed without an update so the current default is fine.

Reply:
So what about the remaining changes in this PR? Were they warnings as well?